x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/use_host_isolation_status.tsx

```diff
@@ -59,7 +59,7 @@ export const useHostIsolationStatus = ({
           return;
         }
 
-        if (isMounted && error.body.statusCode === 404) {
+        if (isMounted && error.body.statusCode === 400) {
```

Comment:
thanks for identifying why we weren't showing the message.

Could we change this to check for `400` or `404` ?  I just want to make sure we don't change logic that another scenario was dependent on